crates/e2e/tests/e2e/liquidity.rs

```diff
@@ -200,6 +200,7 @@ fn create_zeroex_liquidity_orders(
         // fully covers execution costs
         maker_amount: order_creation.buy_amount.as_u128() * 3,
         taker_amount: order_creation.sell_amount.as_u128() * 2,
+        remaining_fillable_taker_amount: order_creation.sell_amount.as_u128(),
```

Comment:
Sorry, I think I misunderstood you earlier. It's indeed not so helpful to test the behavior where the remaining amount is exactly enough to fill the cow order.
Ideally the test case should probably be that the 0x order is partially filled and only a portion of the remaining fillable amount gets used from the order.
That way we can test that scaling original amounts based on the fill state works and that the solver can correctly use only a portion of a 0x order.

Reply:
True, fixed. Made it partially fillable but sufficient to fill the cowswap order.